docs/README.md

```diff
@@ -94,8 +94,49 @@ Download and extract the appropriate binaries from
+| `COR_ENABLE_PROFILING`               | .NET Framework         | `1`                                                                                 |
+| `COR_PROFILER`                       | .NET Framework         | `{918728DD-259F-4A6A-AC2B-B85E1B658318}`                                            |
+| `COR_PROFILER_PATH_32`               | .NET Framework         | `$INSTALL_DIR/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll`                 |
+| `COR_PROFILER_PATH_64`               | .NET Framework         | `$INSTALL_DIR/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll`                 |
+| `CORECLR_ENABLE_PROFILING`           | .NET (Core)            | `1`                                                                                 |
+| `CORECLR_PROFILER`                   | .NET (Core)            | `{918728DD-259F-4A6A-AC2B-B85E1B658318}`                                            |
+| `CORECLR_PROFILER_PATH`              | .NET (Core) on Linux   | `CORECLR_PROFILER_PATH=$INSTALL_DIR/OpenTelemetry.AutoInstrumentation.Native.so`    |
+| `CORECLR_PROFILER_PATH`              | .NET (Core) on macOS   | `CORECLR_PROFILER_PATH=$INSTALL_DIR/OpenTelemetry.AutoInstrumentation.Native.dylib` |
+| `CORECLR_PROFILER_PATH_32`           | .NET (Core) on Windows | `$INSTALL_DIR/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll`                 |
+| `CORECLR_PROFILER_PATH_64`           | .NET (Core) on Windows | `$INSTALL_DIR/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll`                 |
+| `DOTNET_ADDITIONAL_DEPS`             | .NET (Core)            | `$INSTALL_DIR/AdditionalDeps`                                                       |
+| `DOTNET_SHARED_STORE`                | .NET (Core)            | `$INSTALL_DIR/store`                                                                |
+| `DOTNET_STARTUP_HOOKS`               | .NET (Core)            | `$INSTALL_DIR/netcoreapp3.1/OpenTelemetry.AutoInstrumentation.StartupHook.dll`      |
+| `OTEL_DOTNET_AUTO_HOME`              | All versions           | `$INSTALL_DIR`                                                                      |
+| `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE` | All versions           | `$INSTALL_DIR/integrations.json`                                                    |
+
+> Some settings can be omitted on .NET (Core). For more information, see [config.md](config.md#net-clr-profiler).
+
+### Shell scripts
+
+You can install OpenTelemetry .NET Automatic Instrumentation
+and instrument your .NET application using the provided Shell scripts.
+Example usage:
+
+```sh
+export OS_TYPE=linux-glibc
+curl -sSfL https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/main/download.sh -O
+sh ./download.sh
+curl -sSfL https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/main/instrument.sh
+. ./instrument.sh
```

Comment:
What do you think of including the `instrument.sh` and `instrument.ps1` to the `zip` file (so that they do not have to be additionally downloaded)?

Then we could change these 2 lines to 

```sh
. ./otel-dotnet-auto/instrument.sh
```

I would rather address it in a separate PR.

Reply:
After considering loads of different install options, I think instrument.sh is definitely needed and should be in the package.
(since in linux you probably need to configure some service watch, so you can easily add instrument.sh before the real applications wake up call)

Leaves only question about service discovery if custom path was used. I don't have currently any real clues for Linux, why you should have custom path.

Reply:
1. If one uses a custom path then he/she can use `$INSTALL_DIR`
2. I think it would be better if the parameter name would be `$OTEL_DOTNET_AUTO_HOME` -> then one could set it globally. It will be also "unique". 

> instrument.sh is definitely needed and should be in the package

After we do it and make a release, we update the script in README.md to take advantage of it. But no sooner than that (we need the package to be published.